lib/DDG/Spice/Spotify.pm

```diff
@@ -0,0 +1,22 @@
+package DDG::Spice::Spotify;
+
+# ABSTRACT: Music search on Spotify
+
+use strict;
+use DDG::Spice;
+
+spice is_cached => 0;
+spice proxy_cache_valid => "200 1d";
+spice wrap_jsonp_callback => 1;
+
+spice to => 'https://api.spotify.com/v1/search?q=$1&market=$2&limit=35&type=track';
```

Comment:
We could prepend `artist:` to the `q` parameter, but in that case the search wouldn't work properly when searching for tracks (eg "Like A Prayer on Spotify").